source/models/creating-updating-and-deleting-records.md

```diff
@@ -0,0 +1,129 @@
+#### Creating Records
+
+You can create records by calling the `createRecord` method on the store.
+
+```js
+store.createRecord('post', {
+  title: 'Rails is Omakase',
+  body: 'Lorem ipsum'
+});
+```
+
+The store object is available in controllers and routes using `this.store`.
+
+Although `createRecord` is fairly straightforward, the only thing to watch out for
+is that you cannot assign a promise as a relationship, currently.
+
+For example, if you want to set the `author` property of a post, this would **not** work
+if the `user` with id isn't already loaded into the store:
+
+```js
+var store = this.store;
+
+store.createRecord('post', {
+  title: 'Rails is Omakase',
+  body: 'Lorem ipsum',
+  author: store.findRecord('user', 1)
+});
+```
+
+However, you can easily set the relationship after the promise has fulfilled:
+
+```js
+var store = this.store;
+
+var post = store.createRecord('post', {
+  title: 'Rails is Omakase',
+  body: 'Lorem ipsum'
+});
+
+store.findRecord('user', 1).then(function(user) {
+  post.set('author', user);
+});
+```
+
+#### Persisting Records
+
+Records in Ember Data are persisted on a per-instance basis.
+Call `save()` on any instance of `DS.Model` and it will make a network request.
+
+Ember Data takes care of tracking the state of each record for
+you. This allows Ember Data to treat newly created records differently
+from existing records when saving.
+
+By default, Ember Data will `POST` newly created records to their type url.
+
+```javascript
+var post = store.createRecord('post', {
+  title: 'Rails is Omakase',
+  body: 'Lorem ipsum'
+});
+
+post.save(); // => POST to '/posts'
+```
+
+While records that already exist on the backend are updated using the
+HTTP `PUT` verb when saved.
```

Comment:
Since `JSONAPIAdapter` is the new default adapter records are saved with `PATCH`